content/raw/chat/mutual_contacts.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Mutual Contact Requests spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Mutual Contact Requests
+
+## Motivation
+
+To prevent unsolicited messages & invitations and some forms of spam, Users must explicity add each other as contacts.
+
+## Designs
+
+* [designs](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5224%3A0)
+* [Sending contact request](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=725%3A329626)
+* [Receiving contact request](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1637%3A369041)
+* [Pending contact request](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=128%3A28247)
+
+## Use Cases
+
+### Sending a Contact Request
+
+1. User selects a profile and asks application sends contact request
+2. User fills reason for contact request e.g "Say who you are / why you want to become a contact.."
+3. User sends contact request
+4. System notifies user `Contact Request Send`
```

Comment:
should there be notifications for a contact request accepted or rejected?

Reply:
A user should not be notified if a contact requested they have made is rejected.

But yes we should push a notification if a contact request the user has made is accepted.

Reply:
designs for how these notifications should appear IF Status is running and infocus are here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=2090%3A252386  If Status is not infocus e.g. it's minimized then OS notifications should be used.